imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -1662,11 +1662,46 @@ func Test_fdSeek_Errors(t *testing.T) {
 
 // Test_fdSync only tests it is stubbed for GrainLang per #271
```

Comment:
Perhaps this comment should be updated/removed?

Reply:
Obviously. I'll update it. Thanks!